### PR TITLE
feat(aliasing): getImportedType() in model Flle in aliasing

### LINKED ISSUE #889: API: getActualImportTypeName() in model Flle in aliasing
# Feature Request 🛍️

Add below api function to modelFile 

---

**Function: `getActualImportType(String s)`**

This function takes an input string `s`, which may represent an alias for a type. In some contexts, type names are aliased to shorter, more convenient literals. The function's purpose is to determine if `s` is an alias for another type and, if so, return the actual type name it represents. If `s` is not an alias, the function simply returns the original input string.

**Behavior:**
- If `s` is an alias mapped to an actual type name, the function returns the actual type name.
- If `s` is not an alias (i.e., it directly represents the actual type name), the function returns `s` unchanged.

## Use Case

This API has use cases in the concerto-codegen for implementing aliasing also it would make the code clean.

## Context

While working with the codegen I felt the need to create a separate api for above usecase.

### PR DESCRIPTION
**Pull request by salujajaskeerat**
# Closes #889 

This API is useful for developing the concerto-codegen for the aliasing support

### Related Issues
- Issue #889
- Pull Request #<NUMBER>

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [x] Vital features and changes captured in unit and/or integration tests
- [ ] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [x] Extend the documentation, if necessary
- [x] Merging to `main` from `fork:branchname`

### CODE REVIEW

File: packages/concerto-core/lib/introspect/modelfile.js [hunk 1]
```diff
@@ -389,6 +389,18 @@ class ModelFile extends Decorated {
         }),this);
     }
 
+    /**
+     * Returns the actual imported name from another namespace
+     * @param {string} type - the short name of the type
+     * @returns {string} - the actual imported name. If not aliased then returns the same string
+     */
+    getActualImportType(type) {
```
Comment (ekarademir) [inline packages/concerto-core/lib/introspect/modelfile.js:397]: Can you add a unit test for this function please?

Comment (salujajaskeerat) [reply packages/concerto-core/lib/introspect/modelfile.js:397]: added the unit test-cases

Comment (ekarademir) [reply packages/concerto-core/lib/introspect/modelfile.js:397]: let's call this,`getImportedType`

Comment (salujajaskeerat) [reply packages/concerto-core/lib/introspect/modelfile.js:397]: Sure. 👍🏻

**Review by muhabdulkadir — APPROVED** [review]

**Comment by muhabdulkadir** [comment]
Resolve the changes and this should be fine.

**Review by ekarademir — APPROVED** [review]

**Pull request merged**